3_run_analysis.R

```diff
@@ -209,7 +209,7 @@ if (file.exists(bonds_inputs_file)) {
 gather_and_save_project_results(results_path, aggregation_level = "portfolio")
 gather_and_save_project_results(results_path,
   aggregation_level = "company",
-  year_filter = c(start_year, start_year + 5),
+  year_filter = c(start_year, start_year + time_horizon),
```

Comment:
I can't easily tell if/where `time_horizon` comes from and if it will always be available at this point. Not saying it's wrong, just hard for me to verify. I trust your judgement here.

Reply:
it comes form `AnalysisParameters.yml` or `ProjecParameters.yml` (depending on the context. It is available as soon as we run `set_global_parameters()` or `set_project_parameters()`